.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+
+---
+
+/kind bug
```

Comment:
@Fryguy should we change these to @miq-bot commands directly? Like this one could be `@miq-bot add_label bug`

Reply:
Oh nice idea!

Reply:
Is `/kind` a bot thing?

Reply:
Yes, it is a prow command https://prow.k8s.io/command-help